### DISCUSSION TITLE
Version 2.4.0 Cookie Problem?

### DISCUSSION THREAD

**Discussion by JamesSalzer** [thunderclient/thunder-client-support#1005] [Q&A] [answered]
I don't know if it's a bug, that's why I'm writing here.
I have a collection in which I test the routes in our Laravel installation, so far everything has worked wonderfully, today I switched to
Thunder Client v2.4.0 and I get errors when I run my collection.
I searched a bit and found out that the error occurs when I pass cookies in the headers, without cookies it runs through error free.
Is this a bug or my fault?

![ThunderMitFehler](https://user-images.githubusercontent.com/115971524/217522146-b103e55b-e30b-441b-8b69-875861fcd2be.jpg)
![ThunderOhneFehler](https://user-images.githubusercontent.com/115971524/217522151-2a1e2479-6da0-4d1a-b8c7-7a2ef72996b9.jpg)

**Comment by rangav** [comment]
Thanks @JamesSalzer for reporting, will look into it.

Can you restart vscode and try again.

**Comment by JamesSalzer** [comment]
Restart did not bring success, I have now downgraded to version 2.3.4 and everything runs again without problems.

**Reply by rangav** [reply]
I have fixed the bug, please update to `v2.4.1`

**Comment by JamesSalzer** [comment, marked as answer]
Great, it works
